adapters/eplanning/eplanning.go

```diff
@@ -187,6 +227,20 @@ func (adapter *EPlanningAdapter) MakeRequests(request *openrtb.BidRequest, reqIn
 	return requests, errors
 }
 
+func isMobileDevice(request *openrtb.BidRequest) int {
+	if request.Device != nil {
+		if request.Device.DeviceType == openrtb.DeviceTypePersonalComputer {
+			return 0
+		}
+		if request.Device.DeviceType == openrtb.DeviceTypeMobileTablet ||
+			request.Device.DeviceType == openrtb.DeviceTypePhone ||
+			request.Device.DeviceType == openrtb.DeviceTypeTablet {
+			return 1
+		}
+	}
+	return -1
+}
```

Comment:
This method returns 3 states: mobile, desktop, or unknown. However, the sorter only cares if its mobile and groups desktop and unknown together. Consider keeping things simple and returning a boolean here.